t/10_translators.t

```diff
@@ -87,4 +87,16 @@ is (
     'datetime recorded properly from iso8601 (w/ zulu tz)',
 );
 
+my $album = $dbh->album->where( rating => 'fair' )->next;
+is $album->rating->handle, 'fair', 'query by enum handle';
+
+$album = $dbh->album->where( rating => 500 )->next;
+is $album->rating->handle, 'fair', 'query by enum id';
+
+$album->rating('sucks');
+is $album->rating->handle, 'sucks', 'set enum with handle';
+
+$album->rating(600);
```

Comment:
Might be good to also include a test for an invalid integer value here, to make sure it still fails appropriately.  (1. nonexistent enum_id, and 2. existent enum_id but not mapped to that field)

Reply:
Noted.  Pushed a commit to add testing of invalid values.